weblate/lang/tests.py

```diff
@@ -248,6 +248,30 @@ def test_chinese_fuzzy_get(self):
         )
         self.run_create("zh-rCN", "zh_CN", "ltr", "0", "Chinese (zh_CN)", False)
 
+    def test_case_sensitive_fuzzy_get(self):
+        """Test handling of manually created zh-TW, zh-TW and zh_TW languages."""
```

Comment:
This test will fail on MySQL, so it shoud be skipped:

```suggestion
        """Test handling of manually created zh-TW, zh-TW and zh_TW languages."""
        if settings.DATABASES["default"]["ENGINE"] == "django.db.backends.mysql":
            raise SkipTest("Not supported on MySQL")
```

This will need SkipTest and settings imported to this module as well.